<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LPTPortTestDlg</class>
 <widget class="QDialog" name="LPTPortTestDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupData">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Databits:</string>
   </property>
   <widget class="QCheckBox" name="ckData_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckData_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckData_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckData_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckData_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckData_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckData_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckData_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSetData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReadData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>read</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupControl">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>91</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Control:</string>
   </property>
   <widget class="QCheckBox" name="ckControl1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>! line feed</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckControl2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckControl3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckControl4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckControl5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>! strobe</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckControl6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>! select prt</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckControl7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckControl8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSetControl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReadControl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>read</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupStatus">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>91</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Status:</string>
   </property>
   <widget class="QCheckBox" name="chStatus1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chStatus2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>paper-out</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chStatus3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chStatus4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ack</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chStatus5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chStatus6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>error</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chStatus7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chStatus8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>! busy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReadStatus">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>read</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label1_17">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1_18">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ip</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1_19">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1_20">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ei</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1_21">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ds</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1_22">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>lf</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1_23">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1_24">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>si</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPortState">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddress">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>57</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Portaddress:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
